modules/creating-logfilesmetricexporter.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-logfilesmetricexporter_{context}"]
+= Creating a LogFileMetricExporter resource
+
+In the {logging} version 5.8 and newer versions, the `LogFileMetricExporter` custom resource (CR) is no longer deployed with the collector by default. You must manually create a `LogFileMetricExporter` CR to collect container related metrics. If you do not create the `LogFileMetricExporter` CR, you may see a *No datapoints found* message in the {product-title} web console dashboard for *Produced Logs*.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {clo}.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Create a `LogFileMetricExporter` CR as a YAML file:
++
+.Example `LogFileMetricExporter` CR
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogFileMetricExporter
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  nodeSelector: {}
+  resources:
+    limits:
+      cpu: 500m
+      memory: 256Mi
+    requests:
+      cpu: 200m
+      memory: 128Mi
+  tolerations: []
+----
+
+. Apply the `LogFileMetricExporter` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+.Verification
+
+* Verify that the pods have been created in the namespace, by running the following command and observing the output:
```

Comment:
I'll be more specific here. A `logfilesmetricexporter` pod should be running in parallel to where a collector pod is running per node. If this is not happening because not set correctly the tolerations or nodeSelector, then, the metrics from that collector where not `logfilesmetricexporter` pod are not collected